pwa/src/pages/settings/databases/[databaseId]/DatabaseDetailPage.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from "react";
```

Comment:
When I go to a detail page of a database I see in the breadcrumbs `[databaseId]`
This can be changed in the `gatsby-config.js` file. Just add:
```
 {

   pathname: "/databases/[databaseId]",
   crumbLabel: "Database",
 },
```
it's important to do this when creating a new detail page

![image](https://github.com/ConductionNL/gateway-ui/assets/43807324/3ce0b7ff-c42c-4ad9-9fe3-6394964b5af1)